### DISCUSSION TITLE
Truck Box

### DISCUSSION THREAD

**Discussion by winder** [winder/Universal-G-Code-Sender#1532] [Show and tell]
My 2 year old constantly has an arm full of toy cars, so I made him a truck box.

The image was vectorized with inkscape and slightly modified.
Text and CAM was done with Easel
After the truck sides were cut out I winged a toolbox with regular woodworking tools.

![truck1](https://user-images.githubusercontent.com/125509/107124437-eb1be880-6871-11eb-982f-9fbf198bee4c.jpg)
![truck2](https://user-images.githubusercontent.com/125509/107124440-ec4d1580-6871-11eb-93a2-fcef239116fc.jpg)
![truck3](https://user-images.githubusercontent.com/125509/107124443-ee16d900-6871-11eb-9aa5-82a77cae930a.jpg)

**Comment by JacksCrafts** [comment]
Adding the year was a nice touch.
